core/src/main/scala/slamdata/domain/PullRequestDraft.scala

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2020 Precog Data
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package precog.domain
+
+import github4s.domain.{PullRequest, PullRequestBase, User}
+
+case class PullRequestDraft(id: Int,
```

Comment:
Should be `final`. Also class formals should be formatted analogously to method formals: when split across lines, double-indent rather than vertically aligning, with `extends` and `with` being similarly double-indented and on their own line(s).